modules/install-sno-installing-with-usb-media.adoc

```diff
@@ -0,0 +1,53 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="installing-with-usb-media_{context}"]
+
+= Installing with USB media
+
+Installing with USB media involves creating a bootable USB drive with the discovery ISO on the administration node. Booting the server with the USB drive prepares the node to install singe node OpenShift (SNO) via the Assisted Installer (AI).
```

Comment:
s/involves/requires/ ?

nit: I'd replace "via" with "with", but the ISG doesn't forbid the use of "via."